downstream/modules/aap-on-azure/proc-peer-vwan-hub-to-aap.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-peer-vwan-hub-to-aap"]
+
+= Peering a VWAN Hub to the {AAPonAzureNameShort} Network
+
+Before you can peer a VWAN with the {AAPonAzureNameShort} VNET, you must have connected a hub network and at least one spoke network to the hub network of the Azure VWAN that you intend to connect {AAPonAzureNameShort} to.
```

Comment:
```suggestion
Before you can peer a VWAN with the {AAPonAzureNameShort} VNET, you must connect a hub network, and at least one spoke network, to the hub network of the Azure VWAN to which you intend to connect {AAPonAzureNameShort}.
```

Reply:
This reads like a prerequisite to me rather than an abstract. What do we mean by peering a VWAN hub to the network? What is the user motivation for completing this procedure?

Reply:
Peering is explained earlier in the doc. The AAP VNET is isolated when it's created, so it needs to be peered with other networks so that (1) it can be reached and (2) it can reach networks where you want to perform automation. VWAN is one of 3 options for peering.